Azure/azure-service-operator
# Updating Helm Chart and adding custom namespace

Updating Helm Chart. 
* Previously the dependencies charts directory was missing for aad-pod-identity, added that in.
* Updating the chart tar file with newest CRD changes. 
* Updating default image with newest code changes. 
* Added custom namespace support, as well as environment variable for specifying the Azure Cloud Environment. 
* Updated docs with this change

Closes #955

**What this PR does / why we need it**:

**Special notes for your reviewer**:
How to test:
- attempt to install Helm Chart and verify that it works
- pass in a custom namespace and verify that install succeeds, and that the azureoperator-system namespace is not created

**How does this PR make you feel**:
![gif](https://giphy.com/)

**If applicable**:
